.gitattributes

```diff
@@ -0,0 +1 @@
+*.php text eol=lf
```

Comment:
Could you export ignore `.github` ?

Reply:
Hi, what do mean with "export"?

Reply:
Hi
The file controls `git archive` and basically, excluding some files nobody wants on their production servers is always cool. In debian we do not need the github folder for example 

https://github.com/williamdes/mariadb-mysql-kbs/blob/main/.gitattributes#L15
https://salsa.debian.org/phpmyadmin-team/mariadb-mysql-kbs/-/tree/upstream

Tcpdf: https://salsa.debian.org/phpmyadmin-team/tcpdf/-/tree/upstream

Reply:
Thx. I've done that.

Reply:
Thank you!
One note about your commit message it is more "do not export github folder in vendor tarballs" ;)

Reply:
Thanks for the info.